src/LCT.Services/Sw360CreatorService.cs

```diff
@@ -117,7 +117,15 @@ public async Task<FossTriggerStatus> TriggerFossologyProcess(string releaseId, s
             }
             catch (HttpRequestException ex)
             {
-                Logger.Logger.Log(null, Level.Error, $"\tTriggering the FossologyProcess is failed due to {ex.StatusCode} : {ex.Message}", null);
```

Comment:
you can put it in a separate method for fossology and call
From line no 120 to 128